content/community/sig/ml-ai-across-galaxy/index.md

```diff
@@ -0,0 +1,173 @@
+  <slot name="/community/sig/common_linkbox" />
+</div>
+
+# Welcome to **MALIC – Machine Learning and Artificial Intelligence across the Galaxy Community!**  
+*The Galaxy-wide community for ML & AI across disciplines*
+
+**MALIC** is a stylized name representing the key themes of our community:  
+- **M** – Machine  
+- **A** – Artificial  
+- **L** – Learning  
+- **I** – Intelligence  
+- **C** – Community (across Galaxy)
+
+**Like every Galaxy community, we have our own unique flavor!**
+MALIC brings together everything related to machine learning (ML) and artificial intelligence (AI) within Galaxy.  
+We connect users, developers, and trainers from diverse fields to collaborate, share, and grow together in this transformative era of AI.
+
+**Our mission is to connect fields such as** *digital humanities*, *omics*, *ecology*, **and** *imaging*, **by sharing expertise that strengthens the Galaxy platform.**
+
+## Could MALIC be your new home in Galaxy? Let’s find out!
+
+We’re a growing, welcoming community of people using and developing ML/AI tools in Galaxy.  
+Whether you're working with data in biology, ecology, humanities, imaging or anywhere in between, there's a place for you here!
+
+- **Developers**  
+  Are you wrapping or building ML or deep learning tools? Join us to coordinate efforts, avoid duplications and get early feedback from others in the community.
+- **Users**  
+  Are you using Galaxy ML tools for omics, images or texts? Share your experiences, tell us what works and what could be improved. Your feedback helps shape future tool design.
+- **Trainers**  
+  Are you contributing tutorials or running workshops? Help us curate and connect training resources across different use cases and domains.
+- **Researchers**  
+  Are you exploring new ML applications in Galaxy? Join us to benchmark workflows, improve reproducibility and exchange ideas across disciplines.
+
+## MALIC Community Goals
+
+Our primary goals are to:
+* **Coordinate efforts** across the Galaxy ecosystem to reduce redundancy in tools, workflows, and training.
+* **Share our work** from within and outside Galaxy to showcase our advancements.
+* **Support each other** through knowledge-sharing, troubleshooting, and collaboration.
+
+Additionally, we strive to:
+* **Develop and maintain** ML/AI tools within Galaxy.
+* **Implement FAIR principles** to make workflows reproducible and reusable.
+* **Expand documentation** and **training resources** for accessible learning and skill-building.
+
+## Meetings and Communication
+
+- **When?** We meet every third Thursday at 16:00 CET  
+- **How long?** Just one hour, short and sweet!  
+- **What’s on the agenda?** Tool demos, cross-domain updates, collaboration planning, and relaxed idea sharing.
+
+- Our community connects via our dedicated [Matrix channel](https://matrix.to/).
+- Check our [Agenda](googledoc) to find out more about what we are currently doing. 
```

Comment:
Can you update the agenda link below: https://docs.google.com/document/d/1ElFZtbUemUXvygcozbrvS7XEfMIpi0eYPT-gsnjVq3o/edit